### PR TITLE
Allow for no authorizedSubjects array

### PR DESCRIPTION
**Pull request by BCook98**
Matching the ASAP spec
- The resource server MAY decide if the verified issuer is authorised to communicate with the resource server.
- The resource server MAY decide if the verified issuer is authorised to make requests in relation to the claimed subject (principal).
- The resource server MAY decide if the combination of verified issuer and effective subject is authorised to make the requested business operation.

### CODE REVIEW

File: test/unit/server/validator/jwt-claims-validator.spec.js [hunk 1]
```diff
@@ -233,6 +233,15 @@ describe('jwtClaimsValidator', function () {
             .fail(failTest(done));
     });
 
+    it('should accept a valid jwt token with no authoirized subjects array', function(done) {
```
Comment (donnellbob) [inline test/unit/server/validator/jwt-claims-validator.spec.js]: typo - authoirized. very nit picky but yea :P

**Comment by donnellbob** [comment]
5 & 6 of travis build seems to of failed.

**Comment by BCook98** [comment]
Fixed the typo. For some reason the travis builds like to hang and timeout :(

**Comment by donnellbob** [comment]
LGTM YOLO 👍

**Pull request merged**